var/spack/repos/builtin/packages/py-bglibpy/package.py

```diff
@@ -3,20 +3,23 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
+import spack
```

Comment:
I see the point but in spack packaging convention, `import *` is done in all packages. @matz-e : opinion?

Reply:
I'd keep it in line with upstream.

Reply:
Time for upstream to fix things ;-) Anyway, I've reverted to the upstream behavior